docs/features/readonly-variables-and-parameters.md

```diff
@@ -0,0 +1,61 @@
+### High Level Language Proposal:  
+Allow 'readonly' to be used on local variables and on parameters. 
+
```

Comment:
For the same reason we have "const" members of classes and "readonly fields" in classes.  "const" has a *very* specific meaning to the language already that extends far beyond being able to assign to something.